mods/ra/maps/fall-of-greece-1-personal-war/personal-war.lua

```diff
@@ -489,7 +489,7 @@ WorldLoaded = function()
 	Allies = Player.GetPlayer("Allies")
 	USSR = Player.GetPlayer("USSR")
 	BadGuy = Player.GetPlayer("BadGuy")
-	Greece = Player.GetPlayer("Greece")
+	Greece = Player.GetPlayer("England")
```

Comment:
This looks more than a little bogus. Is it still needed now that you're modifying campaign-palettes?

Reply:
I tend to agree. Assuming this is an automatically imported mission you should keep the country `Greece` and just overwrite it's color in .yaml if that is what you are after.

Reply:
I think @Smittytron found that England was the correct player in the imported mission, and that `Greece` was a copy/paste error. Not 100% sure on that though.

Reply:
The original had the green team as England. I didn't import the file myself but took it from a [submitter on forums](https://forum.openra.net/viewtopic.php?f=83&t=20812&p=309462&hilit=counterstrike#p309462), who changed Greece to Allies in multiple missions, and England to Greece in this particular one.

Allies made more sense to me than Greece so I kept it in several missions that I did later; not understanding palettes at the time. The options to fix this are to either add Allies to the palettes, or to change Allies to Greece in several missions.

Reply:
Why not rename the `Greece` variable to `England`? Surely this is the most correct option.